conda-store-server/conda_store_server/_internal/server/views/api.py

```diff
@@ -128,6 +129,30 @@ def paginated_api_response(
     }
 
 
+def deprecated(sunset_date):
```

Comment:
It might be good to accept a `datetime.datetime` object and then format it however you want inside this function. That moves the responsibility of ensuring the right format away from the person using the decorator, to the decorator itself.